### PR TITLE
Document setnull as deletion method for default searchurls.

### PR DESCRIPTION
**Pull request by dllud**
As asked by @bovine3dom at #2327 
Suggestions are welcomed.

### CODE REVIEW

File: src/lib/config.ts [hunk 1]
```diff
@@ -623,6 +623,8 @@ export class default_config {
      * Examples:
      * - When running `open gi cute puppies`, with a `gi` searchurl defined with `set searchurls.gi https://www.google.com/search?q=%s&tbm=isch`, tridactyl will navigate to `https://www.google.com/search?q=cute puppies&tbm=isch`.
      * - When running `tabopen translate en ja Tridactyl`, with a `translate` searchurl defined with `set searchurls.translate https://translate.google.com/#view=home&op=translate&sl=%s1&tl=%s2&text=%s3`, tridactyl will navigate to `https://translate.google.com/#view=home&op=translate&sl=en&tl=ja&text=Tridactyl`.
+     *
+     * [[setnull]] can be used to "delete" the default search engines. E.g. `setnull searchurls.google`.
```
Comment (bovine3dom) [inline src/lib/config.ts:627]: I'm not sure the `[[setnull]]` linking syntax works across files, have you checked?

Comment (dllud) [reply src/lib/config.ts:627]: It is used already like this, check line 803:
https://github.com/tridactyl/tridactyl/blob/24988248627cbb90c5b07ed6e5308134e02376d5/src/lib/config.ts#L803
`colourscheme` is on another file, [excmds.ts](https://github.com/tridactyl/tridactyl/blob/master/src/excmds.ts), the same as `setnull`.

Comment (bovine3dom) [reply src/lib/config.ts:627]: I have no idea if that means that it works :)

I'll check next time I'm at a PC.

Comment (dllud) [reply src/lib/config.ts:627]: Ahahah well, the `colourscheme` one definitely works. I clicked on it in my current Tridactyl installation.
I guess the `setnull` will work too: same syntax, same destination file.

**Pull request merged**